### PR TITLE
5uncs

### PR DESCRIPTION
**Pull request by ethteck**

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀🚀🚀🚀 This PR matches 5 functions (+0.26%) on `us`.

File: src/a5dd0_len_114e0.c [hunk 1]
```diff
@@ -3333,8 +3334,99 @@ void func_80116674(void) {
...
+                bb = (ModelBoundingBox*) get_model_property(model->modelNode, MODEL_PROP_KEY_BOUNDING_BOX);
+                mX = (bb->minX + bb->maxX) * 0.5f;
+                mY = (bb->minY + bb->maxY) * 0.5f;
+                mZ = (bb->minZ + bb->maxZ) * 0.5f;
+                guMtxXFML(mtx, mX, mY, mZ, &mX, &mY, &mZ);
+                model->currentSpecialMatrix = mtx;
+                model->center.x = mX;
+                model->center.y = mY;
+                model->center.z = mZ;
+                model->flags &= 0xFDFF;
```
Comment (pixel-stuck) [inline src/a5dd0_len_114e0.c:3384]: ~0x200

File: src/ad90_len_2880.c [hunk 2]
```diff
@@ -1,73 +1,67 @@
...
+    f32 sinRotX;
+    f32 cosRotX;
+    f32 temp_f24_2;
     f32 temp_f30;
-    f32 deltaY;
-    s32 temp_lo;
+    s32 var_v1;
 
-    if (camera->unk_06 != 0 || camera->unk_08 != 0) {
+    if (camera->unk_06 != 0 || camera->changingMap) {
```
Comment (nanaian) [inline src/ad90_len_2880.c:18]: I'd prefer `isMapTransition` or `isChangingMap`

**Review by nanaian — COMMENTED** [review]
gg

**Pull request merged**